.gitignore

```diff
@@ -3,4 +3,6 @@ dist/
 *.egg-info/
 __pycache__
 .pytest_cache
-setup.bat
\ No newline at end of file
+setup.bat
+.idea
+.vscode
```

Comment:
Ignored files should not present in repository. Either remove .vscode before ignoring it, or remove this change.